docs/core/about.md

```diff
@@ -5,47 +5,45 @@ ms.date: 09/17/2019
 ---
 # About .NET Core
 
-.NET Core has the following characteristics:
+.NET Core is/has:
 
 - **Cross-platform:** Runs on Windows, macOS, and Linux [operating systems](https://github.com/dotnet/core/blob/master/os-lifecycle-policy.md).
-- **Consistent across architectures:** Runs your code with the same behavior on multiple architectures, including x64, x86, and ARM.
-- **Command-line tools:**  Includes easy-to-use command-line tools that can be used for local development and in continuous-integration scenarios.
-- **Flexible deployment:** Can be included in your app or installed side-by-side (user-wide or system-wide installations). Can be used with [Docker containers](docker/introduction.md).
-- **Compatible:** .NET Core is compatible with .NET Framework, Xamarin, and Mono, via [.NET Standard](../standard/net-standard.md).
-- **Open source:** The .NET Core platform is open source, using MIT and Apache 2 licenses. .NET Core is a [.NET Foundation](https://dotnetfoundation.org/) project.
-- **Supported by Microsoft:** .NET Core is supported by Microsoft, per [.NET Core Support](https://dotnet.microsoft.com/platform/support/policy).
+- **Open source:** The .NET Core platform is [open source](https://github.com/dotnet/core), using MIT and Apache 2 licenses. .NET Core is a [.NET Foundation](https://dotnetfoundation.org/) project.
+- **Modern:** It implements modern paradigms like asyncronous programming, no-copy patterns using structs, and resource governance for containers.
+- **High performance:** Uses [hardware intrinsics](https://devblogs.microsoft.com/dotnet/hardware-intrinsics-in-net-core/) and [constant investment](https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-core-3-0/) to improve performance with every release.
```

Comment:
After reading the "hardware intrinsics" words, the "constant investment" reads for a moment as another feature or pattern to write a high performance code in .NET Core.

Reply:
I agree. I didn't like that when I wrote it. Any thoughts on something better?

Reply:
As far as I understand, this paragraph says two things:

- .NET Core gives developers the tools to write high-performance code (that is, hardware intrinsics, `Span<T>`, pipelines, etc)
- the development team continuously improves the performance of .NET Core itself

However, I don't have good thoughts now on how to say that in one paragraph.

Reply:
How is this? https://github.com/dotnet/docs/pull/16680/files#r367078380